packages/ui/@types/react-table.d.ts

```diff
@@ -0,0 +1,126 @@
+import type {
```

Comment:
Could you elaborate on why we need this override?

Reply:
I added the link exaplaining to the top: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-table.
In a nutshell: the types depend upon declaration merging to work well.